clients/shared/table_config.go

```diff
@@ -15,50 +15,31 @@ import (
 	"github.com/artie-labs/transfer/lib/typing/columns"
 )
 
+func isCommentSet(comment string) bool {
+	return !(comment == "" || comment == "<nil>")
```

Comment:
Where is `<nil>` coming from? Are we coercing a null into a string somewhere where we could just change it to an empty string?

Reply:
`<nil>` is a side effect of https://github.com/artie-labs/transfer/blob/master/clients/shared/table_config.go#L102

Reply:
Can we make that code just write an empty string instead?

Reply:
Sure, I'll leave a TODO so we can do it after default values are handled

Reply:
Or change it to `row := make(map[string]*string)`.